packages/web-components/fast-components/src/listbox-option/listbox-option.styles.ts

```diff
@@ -56,10 +64,10 @@ export const optionStyles: FoundationElementTemplate<
         font-size: ${typeRampBaseFontSize};
         height: calc(${heightNumber} * 1px);
         line-height: ${typeRampBaseLineHeight};
-        margin: 0 calc(${designUnit} * 1px);
+        margin: 0 calc((${designUnit} - ${focusStrokeWidth}) * 1px);
         outline: none;
         overflow: hidden;
-        padding: 0 calc(${designUnit} * 2.25px);
+        padding: 0 1ch;
```

Comment:
Is this a pattern we will be following? This would be equal to the width of 1 zero from the parent font?

Reply:
I know this has come up before and I'm the one responsible for it 😅

Is there a better design token that would work here? The original value resolves to `9px` with our default `designUnit` value of 4.

Reply:
@bheston

Reply:
We should look at opportunities to use ch where it makes sense though.